### PR TITLE
List items import/export

### PR DESCRIPTION
**Pull request by protich**
Add ability to import/export list items

### CODE REVIEW

File: include/class.list.php [hunk 1]
```diff
@@ -229,19 +229,27 @@ function getItems($limit=false, $offset=false) {
...
+            $items->filter(array('id' => $val));
+        elseif ($extra)
+            $items->filter(array('extra' => $val));
         else
-            $criteria['value'] = $val;
+            $items->filter(array('value' => $val));
+
 
-         return DynamicListItem::lookup($criteria);
+        return $items->one();
```
Comment (greezybacon) [inline include/class.list.php]: This will throw an exception in `develop-next`. Seems like we should catch it here

File: include/class.list.php [hunk 2]
```diff
@@ -438,6 +446,134 @@ static function getSelections() {
         return $selections;
     }
 
+    function importCsv($stream, $defaults=array()) {
```
Comment (greezybacon) [inline include/class.list.php]: We **really** need to genericize and reuse this code

File: include/class.list.php (hunk 1 above)
Comment (protich) [inline include/class.list.php]: The "temp" commit is temporary. I simply rigged it to work for current import needs.

File: include/class.list.php (hunk 2 above)
Comment (protich) [inline include/class.list.php]: Agreed -- this was from your list management commit on develop-next.

**Comment by protich** [comment]
Ready for review.

**Comment by greezybacon** [comment]
Since this is a new feature, should we rebase against `develop-next`

**Comment by jdelhome3578** [comment]
@protich @greezybacon Was this re-based?

**Pull request closed without merging**